src/setup/Dockerfile

```diff
@@ -20,7 +20,7 @@
 ########################################################################################
 
 # Base image
-  FROM s390x/ubuntu:16.04
+  FROM s390x/ubuntu:latest
```

Comment:
I'm reluctant to go with "latest" in general because it means we're shipping something that doesn't get tested by us, so we could end up with a broken Dockerfile due to an upstream change, and have no idea.

Reply:
Ah okk, I'll reverse this. Better to test and change it